src/main/java/com/oramon/kata/draw/impl/DrawStrategyImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.oramon.kata.draw.impl;
+
+import com.oramon.kata.draw.DrawStrategy;
+
+/**
+ * ClassName
+ * <p>
+ * Created by Alfredo Gil Rubio
+ * <p>
+ * Date: 26/05/17 12:32
+ */
+public class DrawStrategyImpl implements DrawStrategy {
+
+    private String valueAlive;
+    private String valueDeath;
+
+    public DrawStrategyImpl(String valueAlive,String valueDeath)
+    {
+        this.valueAlive=valueAlive;
+        this.valueDeath=valueDeath;
+    }
+
+    public String getSymbolToDraw(int value) {
+        String result;
+        if (value == 1) {
```

Comment:
Try to work with a constant or a public enums to indentify and be able to change the value more easier